### DISCUSSION TITLE
TSC Meeting Agenda 15th Feb 23

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#51] [Meeting Agenda]
Hi All,

We'll have a fairly loose agenda this time round, mostly discussion based rather than updates. Here are a few things I think we should discuss:

 - Outcomes of board meeting and overview of FINOS corporate goals
 - How we organise ourselves around the TSC responsibilities that are forming, see the [minutes from the last meeting for details](https://github.com/finos/technical-steering-committee/blob/master/meetings/2023-01-18.md#tsc-responsibilities)
 - A discussion around the more formal process we'll need to put in place regarding TSC formation, e.g. voting, number of seats per member tier, community or TSC voted seats. Please see this [previous discussion on the topic](https://github.com/finos/community/issues/162)
 - A brief discussion around attendance, and how it relates to the formation process
 - AOB - hackathon plans, OSFF plans, etc 

If you have anything else you'd like to add, please let me know.

**Comment by caradelia** [comment]
If we could add this issue https://github.com/finos/community/issues/226 regarding an Emerging Tech SIG. Also, to consider is review of the SIG contribution and process as it was last updated OCT 2020.

**Reply by ColinEberhardt** [reply]
Thanks @caradelia - that's a very good point. We don't have a set of principles or process for evaluating SIG proposals. Definitely one to discuss.

**Comment by opoupeney** [comment]
FYI, I'll join by phone tomorrow as I will be commuting between meetings at the time of our sync up.
One subject I'd like to add and discuss if time permits is how to help Finos and the GB to explain what can be done with the existing projects. Sort of map that shows where we stand in the different areas (Interop, Reg, Workflows, AI, Visualization, ...). It should help GB to identify where to allocate budget to strategic initiatives and prospects/customers to understand what Open Source brings. I think this is our role to build and maintain this map as a 2023 objective.
